mnefun/_report.py

```diff
@@ -0,0 +1,364 @@
+                assert isinstance(source, dict)
+                analysis = source['analysis']
+                name = source['name']
+                # Load the inverse
+                inv_dir = op.join(p.work_dir, subj, p.inverse_dir)
+                fname_inv = op.join(inv_dir,
+                                    safe_inserter(source['inv'], subj))
+                inv = mne.minimum_norm.read_inverse_operator(fname_inv)
+                fname_evoked = op.join(inv_dir, '%s_%d%s_%s_%s-ave.fif'
+                                       % (analysis, p.lp_cut, p.inv_tag,
+                                          p.eq_tag, subj))
+                if not op.isfile(fname_inv):
+                    print('Missing inv: %s' % fname_inv)
+                elif not op.isfile(fname_evoked):
+                    print('Missing evoked: %s' % fname_evoked)
+                else:
+                    evoked = mne.read_evokeds(fname_evoked, name)
+                    title = '%s["%s"] (N=%d)' % (analysis, name, evoked.nave)
+                    stc = mne.minimum_norm.apply_inverse(
+                        evoked, inv, lambda2=source.get('lambda2', 1. / 9.),
+                        method=source.get('method', 'dSPM'),
+                        pick_ori=None)
+                    stc = abs(stc)
+                    # get clim using the reject_tmin <->reject_tmax
+                    stc_crop = stc.copy().crop(p.reject_tmin, p.reject_tmax)
+                    colormap = source.get('colormap', 'viridis')
+                    transparent = source.get('transparent', True)
+                    clim = source.get('clim', dict(kind='percent',
+                                                   lims=[82, 90, 98]))
+                    clim = mne.viz._3d._limits_to_control_points(
+                         clim, stc_crop.data, colormap)[0]
+                    clim = dict(kind='value', lims=clim)
+                    if not isinstance(stc, mne.SourceEstimate):
+                        print('Only surface source estimates currently '
+                              'supported')
+                    else:
+                        brain = stc.plot(
+                            hemi='split', views=['lat', 'med'],
+                            size=(800, 600), foreground='k', background='w',
+                            colormap=colormap, clim=clim,
+                            transparent=transparent)
+                        imgs = list()
+                        times = source.get('times', [0.1])
+                        for t in times:
+                            brain.set_time(t)
+                            imgs.append(brain.screenshot())
+                        brain.close()
+                        captions = ['%2.3f sec' % t for t in times]
+                        report.add_slider_to_section(
+                            imgs, captions=times, section=section,
+                            title=title, image_format='png')
```

Comment:
Is the slider supposed to allow for scrolling in time?

Reply:
Yes if you give multiple time points it will

Reply:
Ok I see. It's a little clunky but it does work for the default `times=[0.1, 0.2]`.